IHP/IDE/CodeGen/ApplicationGenerator.hs

```diff
@@ -184,10 +186,46 @@ generateGenericApplication applicationName =
                 <> "import " <> applicationName <> ".Routes ()\n"
                 <> "import " <> applicationName <> ".View.Context\n"
                 <> "import Application.Helper.View\n"
+
+            welcomeControllerStaticHs =   
+                "module " <> applicationName <> "Controller.Static\n"
+                <> "import " <> applicationName  <>".Controller.Prelude\n"
+                <> "import " <> applicationName  <>".View.Static.Welcome\n"
+                <> "\n"
+                <> "instance Controller StaticController where\n"    
+                <> "    action WelcomeAction = render WelcomeView\n"
+
+            welcomeViewStaticHs = 
+              "module " <> applicationName <> ".View.Static.Welcome\n"
+             <>"import " <> applicationName <> ".View.Prelude\n"
+             <>"\n"
+             <>"data WelcomeView = WelcomeView\n"
+             <>"\n"
+             <>"instance View WelcomeView ViewContext where\n"
+             <>"    html WelcomeView = [hsx| <div style=\"background-color: #657b83; padding-top: 2rem; padding-bottom: 2rem; color:hsla(196, 13%, 96%, 1)\">\n"
+             <>"    <div style=\"max-width: 800px; margin-left: auto; margin-right: auto\">"
+             <>"        <h1 style=\"margin-bottom: 2rem; font-size: 2rem; font-weight: 300; border-bottom: 1px solid white; padding-bottom: 0.25rem; border-color: hsla(196, 13%, 60%, 1)\">λ IHP</h1>"
+             <>"        <h2 style=\"margin-top: 0; margin-bottom: 0rem; font-weight: 900; font-size: 3rem\">It's working!</h2>"
+             <>"            <p style=\"margin-top: 1rem; font-size: 1.75rem; font-weight: 600; color:hsla(196, 13%, 80%, 1)\">Your new application is up and running.</p>"
+             <>"                <a href=\"https://ihp.digitallyinduced.com/Guide/your-first-project.html\" "
+             <>"                   style=\"margin-top: 2rem; background-color: #268bd2; padding: 1rem; border-radius: 3px; color: hsla(205, 69%, 98%, 1); " 
+             <>"text-decoration: none; font-weight: bold; display: inline-block; box-shadow: 0 4px 6px hsla(205, 69%, 0%, 0.08); transition: box-shadow 0.2s; transition: transform 0.2s;\" tagert=\"_blank\"> "
+             <>"Learn the Next Steps in the Documentation" 
+             <>"                </a>\n"
+             <>"    </div>\n"
+             <>"</div>\n"
+             <>"<div style=\"max-width: 800px; margin-left: auto; margin-right: auto; margin-top: 4rem\">\n"
+             <>"    <img src=\"icon.svg\" alt=\"ihp_icon\">"
```

Comment:
This needs to be an absolute path as e.g. for an `Admin` app, this view would be visible at `http://localhost:8000/admin/`. Then the `icon.svg` would refer to `http://localhost:8000/admin/icon.svg` which would fail.

```suggestion
             <>"    <img src=\"icon.svg\" alt=\"/ihp_icon\">"
```

Maybe we can rename this file to `ihp-welcome-icon.svg`. Otherwise this will conflict when people add some other project specific `icon.svg` to their projects :)